pages/events/[slug].vue

```diff
@@ -1,3 +1,248 @@
+<script setup>
+// COMPONENTS
+import { NavBreadcrumb } from 'ucla-library-website-components'
+
+// HELPERS
+import _get from 'lodash/get'
+
+// GQL
+import FTVAEventDetail from '../gql/queries/FTVAEventDetail.gql'
+
+const { $graphql } = useNuxtApp()
+
+const route = useRoute()
+
+// DATA
+const { data, error } = await useAsyncData(`events-detail-${route.params.slug}`, async () => {
+  const data = await $graphql.default.request(FTVAEventDetail, { slug: route.params.slug })
+
+  // return { data, ftvaEventSeries }
+  return data
+})
+if (error.value) {
+  throw createError({
+    ...error.value, statusMessage: 'Page not found.' + error.value, fatal: true
+  })
+}
+
+if (!data.value.ftvaEvent) {
+  // console.log('no data')
+  throw createError({
+    statusCode: 404,
+    statusMessage: 'Page Not Found',
+    fatal: true
+  })
+}
+
+// const page = ref(_get(data.value.single, 'entry', {}))
+const page = data.value.ftvaEvent
+// Breadcrumbs
+const breadcrumbs = page.title
+// Banner Header
+const imageCarousel = page.imageCarousel
+const eventTitle = page.title
+// CardMeta
+const cardMetaCategory = data.value.ftvaEventSeries[0].title
```

Comment:
use loddash package _get and refs to avoid null or undefined errors in the build and refs to achieve reactivitity in vue